### DISCUSSION TITLE
Invalid file descriptor to ICU data received.

### DISCUSSION THREAD

**Discussion by Joseph-Borriello** [cefsharp/CefSharp#4164] [General]
- **What version of the product are you using?**
    - 103.0.90

- **What architecture x86 or x64?**
    - x64
    
- **On what operating system?**
    -  Win10

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    - OffScreen .Net 3.1
    
- **What steps will reproduce the problem?**
    - Running Cef.Initialize(settings, true, null) crashes the program. Debug.log shows [0713/154750.230:ERROR:icu_util.cc(226)] Invalid file descriptor to ICU data received.

- **What is the expected output? What do you see instead?**
    - Normal Processing
    -
- **Please provide any additional information below.**
    - A stack trace if available, any Exception information.
        - none
        
    - Does the cef log provide any relevant information? (By default there should be a debug.log file in your bin directory)
        - [0713/154750.230:ERROR:icu_util.cc(226)] Invalid file descriptor to ICU data received.
        
    - Any other background information that's relevant? Are you doing something out of the ordinary? 3rd party controls?
       - Azure Function Apps
       
- **Does this problem also occur in the `CEF` Sample Application from http://opensource.spotify.com/cefbuilds/index.html?**
      - Works fine on local machine using same exact logic

**Comment by Joseph-Borriello** [comment]
Evening! I already applied this fix https://github.com/cefsharp/CefSharp/issues/2166 which solved the loading issues. From what I can tell, the only remaining problem is that icudtl.dat file (which exists in the directory is needs to be in).

**Reply by amaitland** [reply]
> Azure Function Apps

https://github.com/projectkudu/kudu/wiki/Azure-Web-App-sandbox#win32ksys-user32gdi32-restrictions

Its my understanding the sandbox in which functions run is too restrictive to run chromium.
